pandas/_libs/tslibs/frequencies.pyx

```diff
@@ -223,7 +223,7 @@ cpdef _period_str_to_code(str freqstr):
     try:
         return _period_code_map[freqstr]
     except KeyError:
-        raise ValueError(INVALID_FREQ_ERR_MSG.format(freqstr))
+        raise ValueError(INVALID_FREQ_ERR_MSG(freqstr))
```

Comment:
This looks like a typo?

Reply:
Hi,
In order to converting str.format() to f string I changed INVALID_FREQ_ERR_MSG definition from predefined string to lambda function. Now error message is defined in one place and is callable without .format().

Current definition:
INVALID_FREQ_ERR_MSG = lambda arg: f"Invalid frequency {arg}"

So now message is called as a function:
INVALID_FREQ_ERR_MSG(arg_str)